### PR TITLE
Fixes #888: ClasspathElementZip.getClasspathElementURI() returns inco…

### LINKED ISSUE #888: ClasspathElementZip.getClasspathElementURI() returns incorrect URIs under Windows
Version 4.8.177

[verbose.log](https://github.com/user-attachments/files/17447938/verbose.log)

When I invoke `ClassgraphElementZip.getClasspathElementURI()` on a JAR file on Windows, I get back: `file:/C:/Users/Gili/Documents/licensed-app/server/target/lib/client.browser-1.0-SNAPSHOT.jar!/index.html`

When I pass this URI to Jetty, it normalizes it to `file:///C:/Users/Gili/Documents/licensed-app/server/target/lib/client.browser-1.0-SNAPSHOT.jar!/` and complains `Base Resource should not be an alias`.

Upon further investigation, it looks like `file:/` is not actually a valid URI format. Is there any reason that ClassGraph is returning `file:/` instead of `file:///`? Is this by design? Or is this a bug? Thank you in advance.

### PR DESCRIPTION
**Pull request by cowwoc**
…rrect URIs under Windows.

* Use file:/// for absolute paths and file: for relative paths.
* Added "Test" suffix to unit tests that were being skipped by Surefire.

### CODE REVIEW

**Comment by cowwoc** [comment]
Any idea why the build is failing on MacOS?

**Comment by lukehutch** [comment]
I created a bug to track the macOS build failure, #890. It's probably a failure to find the Mac version of the Narcissus library (this is not related to your changes, it started happening recently, and I haven't had a chance to look into it yet).

The single windows failure is an example of #810, which again you can ignore (it is going to be tricky to track that one down, since it involves a race condition somewhere, which should be impossible given how ClassGraph was built, but apparently is not...).

File: src/test/java/io/github/classgraph/issues/issue305/Issue305Test.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import io.github.classgraph.ScanResult;
+
+/**
+ * Issue305.
+ */
+public class Issue305Test {
+    private ConsoleHandler errPrintStreamHandler = null;
+    private final Logger rootLogger = Logger.getLogger("");
+
+    /** Reset encapsulation circumvention method after each test. */
```
Comment (lukehutch) [inline src/test/java/io/github/classgraph/issues/issue305/Issue305Test.java:26]: Comment is incorrect (came across via copy/paste from other test)

Comment (cowwoc) [reply src/test/java/io/github/classgraph/issues/issue305/Issue305Test.java:26]: Hmm, for clarification, I don't think I added this comment. I think it was already there.

**Comment by lukehutch** [comment]
Sorry to drop the ball, I thought I merged this already!

**Comment by cowwoc** [comment]
No worries, thank you.

**Pull request merged**